usrp3/tools/scripts/uhd_image_builder_gui.py

```diff
@@ -542,6 +538,27 @@ def grc_populate_list(self, parent, files):
             self.show_not_xml_warning()
             return
 
+    def refresh_oot_dirs(self):
```

Comment:
@primercuervo  see earlier comment- the gui will generate the Makefile.OOT.inc. So uhd_image_builder does not need to be run before the GUI. 

A main difference is that the GUI will parse the Makefile.OOT.inc and then read the Makefile.srcs from each OOT repo "in place" rather than copying the Makefile.srcs content into the E300 or X300 local Makefile.srcs. The implication here is that for OOT modules with an include file, the Makefile.srcs need only include a list of the noc_blocks provided by that repo.